odoo/addons/test_assetsbundle/tests/test_assetsbundle.py

```diff
@@ -569,7 +569,7 @@ def unlink(self):
             return origin_unlink(self)
 
         self.patch(IrAttachment, 'create', create)
-        self.patch(IrAttachment, 'unlink', unlink)
+        self.patch(IrAttachment, '_unlink_attachments', unlink)
```

Comment:
Shouldn't you patch the class `AssetsBundle` instead?

Reply:
True enough :)